seed-server/Dockerfile

```diff
@@ -12,13 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+############# gobuilder
+FROM golang:1.19.3 AS gobuilder
+
+WORKDIR /build
+COPY ./VERSION ./VERSION
+COPY ./.git ./.git
```

Comment:
Why do we need the `.git` folder?

Reply:
Needed for calculation of the effective version stored in the go executable.